.changes/unreleased/tinymce-TINY-10303-2024-01-24.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: Pressing Backspace at the start of an empty `summary` element within a `details`
+  element nested in a list item no longer incorrectly removes the `summary` element.
```

This is long enough already, we can lose that word 😂 
```suggestion
body: Pressing Backspace at the start of an empty `summary` element within a `details`
  element nested in a list item no longer removes the `summary` element.
```